crates/ruff_python_ast/ast.toml

```diff
@@ -193,23 +203,25 @@ fields = [
   { name = "handlers", type = "ExceptHandler*" },
   { name = "orelse", type = "Stmt*" },
   { name = "finalbody", type = "Stmt*" },
-  { name = "is_star", type = "bool" },
+  { name = "is_star", type = "bool", skip_source_order = true },
 ]
 
 [Stmt.nodes.StmtAssert]
 doc = "See also [Assert](https://docs.python.org/3/library/ast.html#ast.Assert)"
 fields = [{ name = "test", type = "Expr" }, { name = "msg", type = "Expr?" }]
+source_order = ["test", "msg"]
 
 [Stmt.nodes.StmtImport]
 doc = "See also [Import](https://docs.python.org/3/library/ast.html#ast.Import)"
 fields = [{ name = "names", type = "Alias*" }]
+source_order = ["names"]
 
 [Stmt.nodes.StmtImportFrom]
 doc = "See also [ImportFrom](https://docs.python.org/3/library/ast.html#ast.ImportFrom)"
 fields = [
   { name = "module", type = "Identifier?" },
   { name = "names", type = "Alias*" },
-  { name = "level", type = "u32" },
+  { name = "level", type = "u32", skip_source_order = true },
```

Comment:
We could make the generator script more clever and decide to automatically skip fields where the type is a primitive (`u32`, `bool`, `Box<str>`). That should remove the need for almost all `skip_source_order` attributes